### PR TITLE
144 - Update Ruby to 3.3.0

### LINKED ISSUE #144: Upgrade to ruby 3.3.0 and bundle update
Final updates before deploy.

### PR DESCRIPTION
**Pull request by haitzlm**
Fixes  #144 was able to deploy this successfully once, then whenever I try to do anything, I get "Your Ruby version is 3.2.2, but your Gemfile specified 3.3.0
"

And of course, my gemfile does NOT say that. Will work through.

### CODE REVIEW

File: config/deploy/qa.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 set :rails_env, :production
 set :bundle_without, %w[development test].join(' ')
-set :branch, 'qa'
+set :branch, 'updateruby'
```
Comment (Janell-Huyck) [inline config/deploy/qa.rb:5]: Don't push a change to the qa deploy branch.

**Comment by haitzlm** [comment]
Done and pushed!


Lisa Haitz
Software Developer 3
Application Development Unit
DTS: UC Libraries IT
University of Cincinnati
513-556-1431
***@***.******@***.***>
She/her/hers

From: Janell Huyck ***@***.***>
Date: Monday, April 1, 2024 at 12:48 PM
To: uclibs/staff-directory-23 ***@***.***>
Cc: Haitz, Lisa (haitzlm) ***@***.***>, Author ***@***.***>
Subject: Re: [uclibs/staff-directory-23] WIP: Updateruby (PR #153)

External Email: Use Caution



Looks like it just needs a rubocop fix atm. The changes look like they should work. :)

File: config/deploy/qa.rb (hunk 1 above)
Comment (Janell-Huyck) [inline config/deploy/qa.rb:5]: It is incorrect to change the branch on QA.  That should remain "qa".  When you make a test deploy of your branch, you can change the branch name LOCALLY on your branch, but that change should not be pushed up to QA.

**Review by Janell-Huyck — CHANGES_REQUESTED** [review]

File: config/deploy/qa.rb (hunk 1 above)
Comment (haitzlm) [reply config/deploy/qa.rb:5]: I changed this back to QA i thought on latest update

**Comment by haitzlm** [comment]
I rebased and ran brakeman, rpsec and rubocop, but i have not redone the database, if that is necessary.

**Review by Janell-Huyck — APPROVED** [review]

**Pull request merged**